src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -31,10 +31,6 @@ class Yaml
     const PARSE_CONSTANT = 256;
     const DUMP_EMPTY_ARRAY_AS_SEQUENCE = 1024;
     const PARSE_KEYS_AS_STRINGS = 2048;
-
-    /**
-     * @experimental in version 3.3
-     */
     const PARSE_CUSTOM_TAGS = 512;
```

Comment:
I think I would then move this flag between `PARSE_CONSTANT` and `DUMP_EMPTY_ARRAY_AS_SEQUENCE`.